modules/ROOT/pages/configure-host.adoc

```diff
@@ -1,21 +1,23 @@
 = Configure and Manage a Host Profile
```

Comment:
Since there is only one host org, please rephrase this as "Configure and Manage the Host Profile"

Reply:
Done!